src/components/shared/snackNotification.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div id="ConnectionStatusSnack-ctn">
+    <v-snackbar v-model="visibility">
+      <span id="snack-message">
+        {{ message }}
+      </span>
+
+      <template #action="{ attrs }">
+        <div
+          v-bind="attrs"
+          id="close-icon"
+          @click="handleClose"
+        >
+          <Button
+            icon="close"
+            color="white"
+          />
+        </div>
+      </template>
+    </v-snackbar>
+  </div>
+</template>
+
+<script>
+import { mapState } from 'vuex'
+import AmbButton from '@/components/shared/Button'
+import { PEER_CONNECTED_NOTIFICATION, PEER_DISCONNECTED_NOTIFICATION, PEER_CONNECTING_NOTIFICATION } from '@/components/utils'
+import { LAST_PEER_CONNECTION_STATUS } from '@/store/mutation-types'
+
+export default {
+  name: 'ConnectionStatusSnack',
+  data: () => ({
+    visibility: false,
+    message: 'Connecting to Ambianic Edge device'
+  }),
+  components: {
+    Button: AmbButton
+  },
+  created () {
+    this.setConnectionStatusNotification()
+  },
+  computed: {
+    ...mapState({
+      peerConnectionStatus: state => state.pnp.peerConnectionStatus,
+      lastConnectionStatus: state => state.pnp.lastPeerConnectionStatus
+    })
+  },
+  methods: {
+    handleClose () {
+      this.visibility = false
+    },
+    setConnectionStatusNotification () {
+      if (this.lastConnectionStatus !== this.peerConnectionStatus) {
+        switch (this.peerConnectionStatus) {
+          case 'PEER_CONNECTING':
+            return this.setNotification(PEER_CONNECTING_NOTIFICATION)
+
+          case 'PEER_CONNECTED':
+            return this.setNotification(PEER_CONNECTED_NOTIFICATION)
+
+          case 'PEER_DISCONNECTED':
+            return this.setNotification(PEER_DISCONNECTED_NOTIFICATION)
+
+          default:
+            break
+        }
+      }
+    },
+    setNotification (message) {
+      this.message = message
+      this.visibility = true
+      this.$store.commit(LAST_PEER_CONNECTION_STATUS, this.peerConnectionStatus)
```

Comment:
@vickywane lets put this flag in a Vuex `statusSnackbar` module to avoid mixing it up with an already complicated pnp state machine. `lastPeerConnectionStatus` is only relevant to the snackbar message. If it is mixed in the pnp module then we need to think about the semantics of it and how to keep it updated each time current peer connection status changes. Unnecessary complication.

Reply:
This alright, however i feel we should only create new modules when we several mutations/actions to perform within each module. 

We only have a single value usecase here

Reply:
So what action do you propose here?

Reply:
I  felt we should reuse an existing vuex module that relates to the state of the PWA. 

However, I created a `SnackbarModule` as requested.